pylint/checkers/spelling.py

```diff
@@ -16,8 +16,12 @@
 
 try:
     import enchant
+    from enchant.tokenize import get_tokenizer, Filter, EmailFilter, URLFilter, WikiWordFilter
 except ImportError:
     enchant = None
+    class Filter:
```

Comment:
I'll try that.

As for coverage data, maybe coveralls only kicks in on passing tests?